goth_tests/domain/ya-provider/test_provider_multi_activity.py

```diff
@@ -14,45 +16,89 @@
 from goth.configuration import load_yaml, Override
 from goth.node import node_environment
 from goth.runner import Runner
+from goth.runner.container.payment import PaymentIdPool
+from goth.runner.container.yagna import YagnaContainerConfig
 from goth.runner.probe import ProviderProbe, RequestorProbe
 
 from goth_tests.helpers.activity import wasi_exe_script, wasi_task_package
 from goth_tests.helpers.negotiation import negotiate_agreements, DemandBuilder
 from goth_tests.helpers.payment import pay_all
 
-logger = logging.getLogger("goth.test.multi-activity")
+logger = logging.getLogger(__name__)
 
 
-@pytest.mark.asyncio
-async def test_provider_multi_activity(
-    common_assets: Path,
-    config_overrides: List[Override],
-    log_dir: Path,
-):
-    """Test provider handling multiple activities in single Agreement."""
+def _topology(assets_path: Path) -> List[YagnaContainerConfig]:
```

Comment:
Like in the case of `test_provider_break_agreement.py`, please use the new way of configuring nodes. This test was already using a config override, let's go back to that.